hip/static/styles/common.scss

```diff
@@ -135,3 +135,16 @@ body.modal-is-open-hip {
 .alert-hip {
   color: $alert;
 }
+
+
+.divider-vertical-hip {
+    display: flex;
+    flex-direction: column;
+}
+
+.divider-vertical-hip:before {
```

Comment:
Since we are modifying `.divider-vertical-hip` with the `before` psuedo-class I think we can nest this class inside of `divider-vertical-hip` using `&:before`.

Reply:
🆗